packages/core/src/components/portal/portal.tsx

```diff
@@ -9,9 +9,11 @@ import * as ReactDOM from "react-dom";
 
 import * as Classes from "../../common/classes";
 import * as Errors from "../../common/errors";
-import { HTMLDivProps, IProps } from "../../common/props";
+import { IProps } from "../../common/props";
 
-export interface IPortalProps extends IProps, HTMLDivProps {
+const isReact15 = typeof ReactDOM.createPortal === "undefined";
```

Comment:
rename this to a more accurate name like `canCreatePortal` or `reactApiHasCreatePortal` or whatever